pkg/controllers/binding/binding_controller.go

```diff
@@ -190,7 +191,7 @@ func (c *ResourceBindingController) newOverridePolicyFunc() handler.MapFunc {
 		}
 
 		bindingList := &workv1alpha2.ResourceBindingList{}
-		if err := c.Client.List(ctx, bindingList); err != nil {
+		if err := c.Client.List(ctx, bindingList, &client.ListOptions{UnsafeDisableDeepCopy: ptr.To(true)}); err != nil {
```

Comment:
Does it make sense to add a `namespace` filter? Like:
```diff
                bindingList := &workv1alpha2.ResourceBindingList{}
-               if err := c.Client.List(ctx, bindingList); err != nil {
+               if err := c.Client.List(ctx, bindingList, &client.ListOptions{UnsafeDisableDeepCopy: ptr.To(true), Namespace: namespace}); err != nil {
```

Reply:
`bindingList`  --> `readonlyBindingList`?

Reply:
> Does it make sense to add a namespace filter

Yes